internal/pkg/githubapi/github.go

```diff
@@ -560,15 +560,41 @@ func isSyncFromBranchAllowedForThisPath(allowedPathRegex string, path string) bo
 	return allowedPathsRegex.MatchString(path)
 }
 
+func isRetriggerComment(body string) bool {
+	return strings.TrimSpace(body) == "/retrigger"
+}
+
+func getPR(ctx context.Context, c *github.PullRequestsService, owner, repo string, number int) (*github.PullRequest, error) {
+	pr, res, err := c.Get(ctx, owner, repo, number)
+	prom.InstrumentGhCall(res)
+	return pr, err
+}
+
 func handleCommentPrEvent(ghPrClientDetails GhPrClientDetails, ce *github.IssueCommentEvent, botIdentity string) error {
 	defaultBranch, _ := ghPrClientDetails.GetDefaultBranch()
 	config, err := GetInRepoConfig(ghPrClientDetails, defaultBranch)
 	if err != nil {
 		return err
 	}
+
+	issue := ce.GetIssue()
```

Comment:
We get some properties from GhPrClientDetails, some from IssueCommentEvent.
We want to potentially remove one in favour of the other.